.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
```

Comment:
https://youtrack.jetbrains.com/issue/IDEA-207303#focus=streamItem-27-3299338.0-0 so I think this comment has the solution to this line appearing

Reply:
Thanks, fixing this